internal/queryset/generator/template.go

```diff
@@ -31,6 +32,15 @@ const qsCode = `
 	  return New{{ .Name }}(db)
   }
 
+	func (qs {{ .Name }}) Select(fields ...{{ $ft }}) {{ .Name }} {
```

Comment:
Fix the indentation please (see `func New{{ .Name }}(db *gorm.DB) {{ .Name }}`)

And it will be great if you also will fix `func (qs {{ .Name }}) w(db *gorm.DB)` indents

Reply:
Fixed the indentation of these files.
Notice that the file contains a mix of tabs and whitespaces, which makes indentation a bit difficult here